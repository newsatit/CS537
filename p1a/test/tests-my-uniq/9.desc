many input files with one invalid file
