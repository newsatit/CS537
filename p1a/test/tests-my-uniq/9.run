./my-uniq INPUT_DIR/9a.in INPUT_DIR/9b.in INPUT_DIR/9c.in
