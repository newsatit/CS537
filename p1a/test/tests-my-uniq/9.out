This line should only appear once
my-uniq: cannot open file
